__tests__/index.test.js

```diff
@@ -166,3 +165,69 @@ describe("GET /api/articles", () => {
       });
   });
 });
+describe('GET /api/articles/:article_id/comments', () => {
+  it('GET 200: should respond with an array of comments for the given article_id, where each comment should have the following properties: comment_id, votes, created_at, author, body, article_id', () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        const {comments} = body
+        const actual = comments;
+        const expected = [
+      {
+        comment_id: 15,
+        body: "I am 100% sure that we're not completely sure.",
+        article_id: 5,
+        author: 'butter_bridge',
+        votes: 1,
+        created_at: '2020-11-24T00:08:00.000Z'
+      },
+      {
+        comment_id: 14,
+        body: 'What do you see? I have no idea where this will lead us. This place I speak of, is known as the Black Lodge.',
+        article_id: 5,
+        author: 'icellusedkars',
+        votes: 16,
+        created_at: '2020-06-09T05:00:00.000Z'
+      }
+    ]
+        expect(actual).toMatchObject(expected);
```

Comment:
It would be a more flexible test to first check the length of the returned array, and then loop through the array checking the general shape of objects in the returned array of objects 🔍